src/main/kotlin/no/nav/helse/prosessering/v1/PdfV1Generator.kt

```diff
@@ -41,6 +41,9 @@ internal class PdfV1Generator  {
             registerHelper("eq", Helper<String> { context, options ->
                 if (context == options.param(0)) options.fn() else options.inverse()
             })
+            registerHelper("skalJobbeEq", Helper<SkalJobbe> { context, options ->
```

Comment:
Kunne man her bare brukt den generelle "eq" funksjonen over? Så slipper vi to metoder som egentlig gjøre samme.

Reply:
Tidligere når feltet skalJobbe var String, førte det til feil når når både en String og en Enum skulle passes gjennom. 
Kan se om det finnes en bedre måte å løse det på :)

Reply:
Aha okei rart

Reply:
eq helperen leser kun String. Så når den prøver å rendre en Enum er det ikke rart at den feiler :p 
Legger på en String verdi på enumen SkalJobbe som sendes inn til templaten. Dermed slipper vi en ekstra helper.

Reply:
Aha sånn ja. Trodde skal_jobbe ble satt til String i mappen, men ser den tar Any. Yes looks good.